cpp/build-support/run_cpplint.py

```diff
@@ -28,6 +28,7 @@
 
 _filters = '''
 -whitespace/comments
+-readability/casting
```

Comment:
This cpplint rule raises spurious warnings about expressions like `array::int32(op)`. I looked at the implementation and it would require patching cpplint to fix